### ISSUE TITLE
Router and a Route does not take an effect

### ISSUE THREAD

**Issue by alperozisik** [smartface/router#6]
Code is following:

```javascript
const Router = require("@smartface/router/src/native/NativeRouter");
const Route = require("@smartface/router/src/router/Route");

const router = Router.of({
    path: "/",
    isRoot: true,
    routes: [Route.of({
        path: "/pages/page1",
        build: (match, state, router, view) => {
            let Page1 = require("pages/page1");
            return view || new Page1(router);
        }
    })]
});

router.push("/pages/page1");
```

**Comment by alperozisik** [comment]
I think it is the same reason here:

```javascript
const router = Router.of({
    path: "/",
    isRoot: true,
    routes: [
        StackRouter.of({
            path: "/pages2",
            routes: [Route.of({
                path: "/pages/page1",
                build: (match, state, router, view) => {
                    let Page1 = require("pages/page1");
                    return view || new Page1(router);
                }
            })]
        })
    ]
});

router.push("/pages/page1");
```

**Comment by cenkce** [comment]
it solved in 1.0.0-beta.1
